packages/lib-core/src/store/PluginStoreContext.tsx

```diff
@@ -31,3 +31,19 @@ export const usePluginStore = (): PluginConsumer & PluginManager => {
 
   return store;
 };
+
+/**
+ * React hook that provides access to a feature flag.
+ *
+ * @example
+ * ```ts
+ * const [flag, setFlag] = useFeatureFlag('MY_FLAG');
+ * ...
+ * setFlag(true);
+ * ```
+ */
+export const useFeatureFlag = (flagName: string): [boolean, (newValue: boolean) => void] => {
+  const store = usePluginStore();
+
+  return [store.getFeatureFlag(flagName), store.setFeatureFlag(flagName)];
```

Comment:
```ts
const currentValue = pluginStore.getFeatureFlag(name);

const setValue = React.useCallback((value: boolean) => {
  pluginStore.setFeatureFlag(name, value);
}, [pluginStore]);

return [currentValue, setValue];
```

Also, should the hook react to changes in feature flags? If so, we'll need some way to subscribe to `PluginStore` for changes in flags so we can re-render the component with an up-to-date `currentValue`. cc @christianvogt 

In other words, assuming `FOO` flag is currently not `true` (or undefined) and you do this in your component:
```ts
const [fooFlag, setFooFlag] = useFeatureFlag('FOO');
setFooFlag(true);
```
then your component should re-render, with `fooFlag` having the value `true`.